From: Tzafrir Cohen <tzafrir@debian.org>
Subject: Adjust the path for console_helper
Forwarded: no

Index: xamarin-pkg-mock/mock/etc/consolehelper/mock
===================================================================
--- xamarin-pkg-mock.orig/mock/etc/consolehelper/mock
+++ xamarin-pkg-mock/mock/etc/consolehelper/mock
@@ -1,5 +1,5 @@
 USER=root
-PROGRAM=/usr/libexec/mock/mock
+PROGRAM=/usr/sbin/mock
 SESSION=false
 FALLBACK=false
 KEEP_ENV_VARS=COLUMNS,SSH_AUTH_SOCK,http_proxy,ftp_proxy,https_proxy,no_proxy
